pkg/placement/monitoring/metrics.go

```diff
@@ -18,13 +18,33 @@ var (
 		"placement/hosts_total",
 		"The total number of active hosts reported to placement service.",
 		stats.UnitDimensionless)
+	actorTypesTotal = stats.Int64(
+		"placement/actortypes_total",
+		"The total number of actor types reported to placement service.",
+		stats.UnitDimensionless)
+	nonActorHostsTotal = stats.Int64(
+		"placement/nonactorhosts_total",
+		"The total number of non actor hosts reported to placement service.",
+		stats.UnitDimensionless)
+
+	nilKey = []tag.Key{}
 )
 
 // RecordActiveHostsCount records the number of active hosts
 func RecordActiveHostsCount(count int) {
 	stats.Record(context.Background(), activeHostsTotal.M(int64(count)))
 }
 
+// RecordActiveActorTypesCount records the number of active actor types
+func RecordActiveActorTypesCount(count int) {
+	stats.Record(context.Background(), actorTypesTotal.M(int64(count)))
+}
+
+// RecordActiveNonActorHostsCount records the number of active non actor hosts
+func RecordActiveNonActorHostsCount(count int) {
```

Comment:
We may need to rename this method  name without `Active` as well.

Reply:
fixed now